README.adoc

```diff
@@ -48,31 +48,31 @@ If you're just starting out with OptaPlanner and want to contribute,
 take a look at our https://issues.redhat.com/issues/?jql=project%20%3D%20PLANNER%20AND%20status%20in%20(Open%2C%20Reopened)%20AND%20labels%20%3D%20starter%20ORDER%20BY%20priority%20DESC[starter issues].
 They're specifically chosen to be easier for first time contributors.
 
-== Developing Drools, OptaPlanner and jBPM
+== Contributing to OptaPlanner
 
-*If you want to build or contribute to a kiegroup project, https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/main/README.md[read this document].*
+Seeking how to contribute for a mutual benefit? That's Great! We have prepared guidelines to save some time and make it easier for you to act in the best interests of everyone.
 
-*It will save you and us a lot of time by setting up your development environment correctly.*
-It solves all known pitfalls that can disrupt your development.
-It also describes all guidelines, tips and tricks.
-If you want your pull requests (or patches) to be merged into main, please respect those guidelines.
+=== Workflow
 
-=== Code style
+- https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/main/README.md#team-workflows[new-issue]: Create https://issues.redhat.com/projects/PLANNER/issues[Jira] ticket
+- https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/main/README.md#source-control-with-git[contribute-source]: Use _GitHub Pull Request_, from your own *fork*
+- https://github.com/kiegroup/droolsjbpm-build-bootstrap/tree/main/.ci[pass-ci-checks]: CI automatically will test build on PR submission. All PR checks should end up with *success*
```

Comment:
I don't think that the link for the CI source code is necessary; hardly any external contributor is interested in that. Let's keep the amount of information in this readme minimal.